Duplicate identifier in v4

Hi,

I'm getting the following error at compile time:

    ERROR in node_modules/nativescript-exoplayer/videoplayer.d.ts(25,3): error TS2300: Duplicate identifier 'observeCurrentTime'.
    node_modules/nativescript-exoplayer/videoplayer.d.ts(84,3): error TS2300: Duplicate identifier 'observeCurrentTime'.

I can see in the `videoplayer.d.ts` file that the `Video` class declares both a property called `observeCurrentTime` and a method.

This bug is currently preventing compilation.

Many thanks!
Josh

--- a/src/videoplayer.d.ts
+++ b/src/videoplayer.d.ts
@@ -78,12 +78,6 @@ export declare class Video extends View {
   getCurrentTime(): number;
 
   /**
-   * Boolean to determine if observable for current time is registered.
-   * @param {boolean} observeCurrentTime - True to set observable on current time.
-   */
-  observeCurrentTime(observeCurrentTime: boolean): void;
-
-  /**
    * Observable for current time of the video duration in milliseconds.
    * @returns {number} Current time of the video duration.
    */

